grunttasks/scan-email-translations.js

```diff
@@ -0,0 +1,85 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  grunt.registerTask('scan-email-translations', 'Scan the generated emails for invalid style tags', [
+    'generate-valid-styles',
+    'verify-styles'
+    ]
+  );
+
+  grunt.registerMultiTask('verify-styles', 'Verify whether the translated styles are valid', function (){
+    var fileArray = this.files[0].src;
+    var contents = grunt.file.read(fileArray.shift());
+    var styleArray = JSON.parse(contents);
+
+    var hasErrors = 0;
+    var styleRegex = /style='(.*)'/ig;
```

Comment:
Not sure this RegExp will work. I think it's too greedy.
I tried reversing the order of the attributes in the `<a>` tag and my .tmp/validStyles.json file looks like:
```
["style='color: #0095dd; text-decoration: none;' href='https://support.mozilla.org'"]
```